content/20.pharmaceuticals.md

```diff
@@ -651,6 +653,20 @@ Interest in TCZ as a treatment for COVID-19 was supported by two meta-analyses t
 Additionally, different studies used different dosages, number of doses, and methods of administration; ongoing research may be needed to optimize administration of TCZ [@doi:10.1016/j.jcv.2020.104443], although similar results were reported by one study for intravenous and subcutaneous administration [@doi:10/d2pk].
 Clinical trials that are in progress are likely to provide additional insight into the effectiveness of this drug for the treatment of COVID-19 along with how it should be administered.
 
+##### Anakinra
+**Anticipated Mechanism.**
+Anakinra is a recombinant of IL-1 receptor antagonist (IL-1Ra) that blocks binding of IL-1α, or IL-1β to IL-1R1.
+Its main task is to reduce inflammation caused by IL-1-mediation, which finds a broad spectrum of applications, including acute stroke [@doi10.1136/jnnp.2004.054882], rheumatoid arthritis [@doi:10.3899/jrheum.090074], and several hereditary, systemic or local inflammatory diseases [@doi:10.1056/NEJMoa055137, @doi:10.1186/s12969-017-0147-3, @doi:10.3389/fphar.2018.01157].
+The application of Anakinra features a very safety profile, and has a very low rate of side effects and prolonged duration.
```

Comment:
Wow, it's quite hard to find good safety profiles of Anakinra! I will keep looking.